### PR TITLE
Formatting and simplifying the README

### PR DESCRIPTION
**Pull request by IshaanDesai**
Following up the in-person discussion with @HelenaKschidock with this PR to modify and simplify the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,207 +24,237 @@ After following the steps below, your repository structure should look like this
...
-```
+
+```bash
 cd dumux
 git clone -b v1.0.0 https://github.com/precice/dumux-adapter.git
 ./dune-common/bin/dunecontrol --only=dumux-adapter/dumux-precice all
 ```
 
-### Additional DuMuX Moduls 
+### Additional DuMux Modules
```
Comment (HelenaKschidock) [inline README.md:84]:
```suggestion
### Additional DuMuX Modules
```

File: README.md [hunk 2]
```diff
@@ -29,207 +24,237 @@ After following the steps below, your repository structure should look like this
...
 
-```
+```bash
 git clone -b develop https://github.com/precice/micro-manager
 cd micro-manager
 pip install --user .
 ```
 
-### DuMuX 
+### DuMux
```
Comment (HelenaKschidock) [inline README.md:47]:
```suggestion
### DuMuX
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
 |-- dumux
-        |-- dumux
+    |-- dumux
         |-- dumux-adapter
-                        |--examples
-                                    |--macro-micro
-                                    ...
-                        ...
+            |--examples
+            |--macro-micro
```
Comment (HelenaKschidock) [inline README.md:16]: for now, `macro-micro` is still being set up in the examples folder (we could change this but would have to do so consistently

File: README.md [hunk 4]
```diff
@@ -29,207 +24,237 @@ After following the steps below, your repository structure should look like this
...
 
 3. Reconfigure and build DuMuX via dunecontrol.
-```
+
+```bash
 ./dune-common/bin/dunecontrol --opts=./dumux/cmake.opts all
 ```
 
-### Setup of the DuMuX Macro-micro example
+## Setup of The macro-micro Example
```
Comment (HelenaKschidock) [inline README.md:112]:
```suggestion
## Setup of the macro-micro example
```

**Review by HelenaKschidock — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (IshaanDesai) [reply README.md:16]: Lets keep it in the examples folder for now, so basically not change this.

File: README.md [hunk 5]
```diff
@@ -29,207 +24,237 @@ After following the steps below, your repository structure should look like this
...
 cd build-cmake/examples/macro-micro/coupled-heat-conduction
 python3 run-micro-manager.py
-``` 
-or in parallel via 
 ```
+
+or in parallel via
+
+```bash
 mpirun -n <num_procs> python3 run-micro-problems.py
```
Comment (HelenaKschidock) [inline README.md:247]:
```suggestion
mpirun -n <num_procs> python3 run-micro-manager.py
```

**Review by HelenaKschidock — APPROVED** [review]

**Pull request merged**